README.md

```diff
@@ -1,2 +1,28 @@
-# Intelligent_placer
-University assignment: python lib that check if items in photo can be placed into figure nearby items
+# Intelligent placer
+Given a picture of several objects on a light horizontal surface and a polygon as input, determine whether it is possible to place all these objects simultaneously on the plane so that they fit into the polygon.
+
+## Task definition
+Algorithm input: 
+- The path to a file with tif, png or jpg extension containing a three-channel RGB image on which on the horizontal light surface there is a sheet of A4 paper with a polygon drawn on the left and objects from a known set on the right;
+- The path to the file with the same extension as in the previous paragraph, which contains an image of the surface on which a sheet of paper with a polygon and items will be placed;
+- 4 integers describing the area within which the integer objects we are going to consider are contained. the first and second numbers represent the start and end pixel (inclusive) horizontally, and the second and third numbers represent the start and end pixel of the area vertically.
+
+Algorithm output:
+- True if goods can be placed in a polygon, otherwise false.
```

Comment:
As for me, word _goods_ isn't suitable in this case. I think, it's worth using _items_ or _objects_

Reply:
Fixed